src/core/MenuSelect/index.tsx

```diff
@@ -63,6 +63,11 @@ export default function MenuSelect<
     InputBaseProps = {},
   } = props;
 
+  // eslint-disable-next-line no-console
+  console.warn(
+    "Warning: MenuSelect will be deprecated and replaced with <DropdownMenu />"
```

Comment:
this gets called with every re-render, which generates lots of noise in the console. Can we only log it once please? 🙆‍♂️ 

e.g.,

```ts
let hasWarned = false;

export default function MenuSelect() {
  if (!hasWarned) {
    console.warn(...)
    hasWarned = true;
  }
}
```

<img width="730" alt="Screen Shot 2022-05-17 at 4 43 01 PM" src="https://user-images.githubusercontent.com/6309723/168929615-7634fb65-da20-4e24-b54d-601a4eb02479.png">

Reply:
Cool,
Warnings for the `DropdownMenu` and `Button` components have been updated to only appear once. 🗡️